### PR TITLE
Create PPQ505, Plant and Plant Product Declaration

### PR DESCRIPTION
**Pull request by rhofvendahl**
This PR creates a schema for PPQ 505, Plant and Plant Product Declaration. This schema also encompasses 505b.

For reference:
https://www.aphis.usda.gov/library/forms/pdf/ppq505.pdf
https://www.aphis.usda.gov/library/forms/pdf/ppq505b.pdf

### CODE REVIEW

File: docs/openapi/components/schemas/common/LaceyActProductDeclaration.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+$linkedData:
+  term: LaceyActProductDeclaration
+  '@id': https://w3id.org/traceability#LaceyActProductDeclaration
+title: Lacey Act Product Declaration
+description: The declaration for a product listed in the Lacey Act, which bans trade in agricultural goods that are illegally taken, possessed, transported or sold.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/LaceyActProductDeclaration.yml:5]:
```suggestion
description: The declaration for a product listed in the Lacey Act, which bans trade in agricultural goods that are illegally taken, possessed, transported, or sold.
```

File: docs/openapi/components/schemas/common/LaceyActProductDeclaration.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  articleOrComponent:
+    title: Article / Component of Article
+    description: A brief description of the article, or component of an article, that is manufactured from plants or plant parts.
+    type: string
+    $linkedData:
+      term: articleOrComponent
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#procedureCode
+  plantScientificNames:
+    title: Plant Scientific Names
+    description: The scientific name (species and genus) for the article / component. If the species of plant used to produce the product varies,or if the species used to produce the product is unknown, enter each species that may have been used to prode the product.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/LaceyActProductDeclaration.yml:42]:
```suggestion
    description: The scientific name (species and genus) for the article / component. If the species of plant used to produce the product varies, or if the species used to produce the product is unknown, enter each species that may have been used to produce the product.
```

Comment (rhofvendahl) [reply docs/openapi/components/schemas/common/LaceyActProductDeclaration.yml:42]: Thanks, fixed

**Review by nissimsan — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]
LGTM

**Review by BenjaminMoe — APPROVED** [review]

**Pull request merged**